City-Bureau/city-scrapers-cle
# 🕷️ Fix spider: Greater Cleveland Regional Transit Authority

## What's this PR do?

Fixes our Greater Cleveland Regional Transit Authority spider (aka. `cle_transit`).

## Why are we doing this?

The spider broke due to changes on the pages it's targeting. The agency now renders much of its calendar content using JS so we use `scrapy-playwright` to parse the DOM. 

## Steps to manually test

After installing the project using `pipenv`:

1. Activate the virtual environment:
```
pipenv shell
```
2. Run the spider:
```
scrapy crawl cle_transit -O test_output.csv
```
3. Monitor the stdout and ensure that the crawl proceeds without raising any errors. Pay attention to the final status report from scrapy.

4. Inspect `test_output.csv` to ensure the data looks valid. I suggest opening a few of the URLs under the source column of test_output.csv and comparing the data for the row with what you see on the page.

## Are there any smells or added technical debt to note?
